### PR TITLE
Basic Event Manager

### PR DESCRIPTION
**Pull request by Duffman9205**
Events currently consist of an (arbitrary) string name and no other data.
See AnimationSystem for a sloppy example that it actually works.

### CODE REVIEW

File: DirtyGame/Game/Core/Systems/Render/AnimationSystem.cs [hunk 1]
```diff
@@ -20,12 +21,21 @@ class AnimationSystem : EntitySystem
...
         {
-
+            // Register the EventCallback function with the EventManager
+            EventManager.Instance.AddListener("EntityAdded", EventCallback);
         }
 
         public override void OnEntityAdded(Entity e)
         {           
-            
+            Event evt = new Event();
```
Comment (sturmE) [inline DirtyGame/Game/Core/Systems/Render/AnimationSystem.cs:30]: I am assuming all of this is just test code. If so, probably dont need to commit it.

**Comment by sturmE** [comment]
Looks good.

**Comment by sturmE** [comment]
O I just noticed this. Dont put anything in EntityFramework project.

**Comment by Duffman9205** [comment]
Oops, sorry I forgot to ask about that. I wasn't sure where to put it.

**Pull request merged**